packages/backend-tasks/README.md

```diff
@@ -22,7 +22,7 @@ const scheduler = TaskScheduler.fromConfig(rootConfig).forPlugin('my-plugin');
 
 await scheduler.scheduleTask({
   id: 'refresh_things',
-  frequency: Duration.fromObject({ minutes: 10 }),
+  cadence: '*/5 * * * *', // every 5 minutes
```

Comment:
Hmm, that's actually an excellent idea. It could even fold the V1 and V2 task settings as well, that'll take a little time for me to make sure that it's properly getting tested and all, but I'll get that updated 👍

Reply:
In retrospect with your comments below, do we want to support the frequency periods long term or make the shift to crontab syntax after a while?

Reply:
I personally find crontab syntax overly technical and a bit of a nuisance, and if it's not something you do regularly you'll have to always google the oddball syntax just to do something super simple. "Hmmm how many asterisks did I have to write in a long line again?" :)

I can absolutely appreciate that the additional power that it allows can be useful, so I still think that adding this ability is a good thing! But my hunch is that very few people actually need that power, and they just want to say "do this roughly every fifth minute". They don't think in terms of schedule, they think in terms of frequency.

Reply:
Yeah I think you're right for the most part. That said, as someone who comes from a mainly low level systems administration background, crontab is what many of my personal colleagues here are most accustom to for scheduling 😢.   In reality though, the additional ability added here was mainly just for the FactRetrievers in the tech-insights system of course haha.